client-ui/src/app/store/actions/dashboard-filters.actions.ts

```diff
@@ -14,15 +14,18 @@
  * limitations under the License.
  */
 
-import { Action } from '@ngrx/store';
+import {Action} from '@ngrx/store';
 
-export enum DashboardFormActionTypes {
-    DASHBOARD_FORM_INITIALIZE = '[Dashboard Form] Initialize form',
+export enum DashboardFilterActionTypes {
+  SET_DATE_RANGE = '[Dashboard] Set date range',
 }
 
-export class InitializeForm implements Action {
-    public readonly type = DashboardFormActionTypes.DASHBOARD_FORM_INITIALIZE
-    constructor(public payload: any) { }
+export class SetDateRange implements Action {
+  public readonly type = DashboardFilterActionTypes.SET_DATE_RANGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected member-variable-declaration: 'type' to have a typedef](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4491035)